### PR TITLE
source-free-agent-connector contribution from craigbloodworth

### PR DESCRIPTION
**Pull request by craigbloodworth**
Auto-generated PR by the Connector Builder for source-free-agent-connector

### CODE REVIEW

**Comment by natikgadzhi** [comment]
Thank you for this! <3 

Just needs the svg icon and a quick fix of allowedHosts in the metadata file. Can you upload the icon? I can take care of the rest.

**Comment by craigbloodworth** [comment]
Sure thing @natikgadzhi 
FreeAgent_Icon_F_Wings_Positive_0069bf
FreeAgent_Icon_F_Wings_Positive_0069bf

File: airbyte-integrations/connectors/source-free-agent-connector/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+metadataSpecVersion: "1.0"
+data:
+  allowedHosts:
+    hosts:
+      - "*"
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-free-agent-connector/metadata.yaml:5]:
```suggestion
      - "api.freeagent.com"
```

**Review by natikgadzhi — APPROVED** [review]
This is shaping up great! I will tweak the allowedHosts thing, run formatters, and merge tonight.

**Pull request merged**